java/statestore/src/main/java/sleeper/statestore/transactionlog/DynamoDBTransactionLogStateStore.java

```diff
@@ -36,11 +39,35 @@ public DynamoDBTransactionLogStateStore(
 
     public static TransactionLogStateStore.Builder builderFrom(
             InstanceProperties instanceProperties, TableProperties tableProperties, AmazonDynamoDB dynamoDB, AmazonS3 s3) {
-        return builder()
+        return builderFrom(instanceProperties, tableProperties, dynamoDB, s3, new Configuration());
```

Comment:
It looks like this method probably shouldn't be here any more? The state store factory should use its own Configuration object.